src/amo/components/CollectionAddAddon/index.js

```diff
@@ -105,31 +139,33 @@ export class CollectionAddAddonBase extends React.Component<InternalProps> {
         username: collection.authorUsername,
       }),
     );
-    this.props.setUIState({ addonWasAdded: false });
-  };
-
-  resetMessageStatus = () => {
-    this.props.setUIState({
-      addonWasAdded: false,
-    });
+    this.resetMessageStatus();
   };
 
   render() {
     const { errorHandler, i18n, uiState } = this.props;
 
     return (
-      <div className="CollectionAddAddon">
+      <Card className="CollectionAddAddon">
         {errorHandler.hasError() ? (
           errorHandler.renderError()
         ) : (
           <TransitionGroup className="CollectionAddAddon-noticePlaceholder">
-            {uiState.addonWasAdded && (
+            {uiState.completedAddonAction && (
               <CSSTransition
                 classNames="CollectionAddAddon-noticePlaceholder-transition"
                 timeout={MESSAGE_FADEOUT_TIME}
               >
-                <Notice type="success">
-                  {i18n.gettext('Added to collection')}
+                <Notice
+                  type={
+                    uiState.completedAddonAction === addonAddedAction
+                      ? 'success'
+                      : 'generic'
+                  }
+                >
+                  {uiState.completedAddonAction === addonAddedAction
+                    ? i18n.gettext('Added to collection')
+                    : i18n.gettext('Removed from collection')}
```

Comment:
when I remove an item in smaller screens (where the search sits below), for some reason it scrolls back to the top of the screen after the message fades...the 'Added to collection' does not do this. Do you see this too?

Reply:
sure thing!